devtools-ci-index.yaml

```diff
@@ -4543,7 +4560,15 @@
             ci_cmd: '/bin/bash tests/.infra/centos-ci/cico_build.sh'
             branch: 'cico-nightly-test'
             build_branch: 'cico-nightly-test'
-            timeout: '3h'       
+            timeout: '3h'    
+        - '{ci_project}-{git_repo}-che-pullrequests-test-temporary':
+            git_organization: eclipse
+            git_repo: che
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash tests/.infra/centos-ci/cico_pr_test.sh'
+            branch: 'cico-pr-test'
+            build_branch: 'cico-pr-test'
```

Comment:
These two properties are imo unecessary (are not used anywhere)

Reply:
I want to start script from `cico-pr-test` branch.